src/Provider/Provider.Web/Views/Dashboard/Dashboard.cshtml

```diff
@@ -2,6 +2,10 @@
 
 @model DashboardViewModel
 
+@{
+    var dashboardSearchEndpointPrefixUrl = @Url.RouteUrl(RouteNames.VacanciesSearchSuggestions_Get, new { term = "" }, Context.Request.Scheme);
```

Comment:
`null` is used instead of `new { term = "" }` in Vacacnies.cshtml. Do we need to specify the term?